### PR TITLE
Added code feedback from mentor

### PR DESCRIPTION
**Pull request by kathrynljackson**
# Iteration Number
0/1

## Link to Iteration
- [x] Did you check all of the functionality of the webpage
- [x] Do both parties understand every line of code
- [ ] Is the code DRY/SRP

## Questions
* Could we make the rest of the page relative to the window or to the sidebar?

## Notes
* After discussing with the group, KJ reached out to her mentor for feedback on the DTR and code.
* The mentor said the DTR looked "great".
* The mentor noticed that the sidebar was not formatted correctly. She suggested making it a **fixed:position** and adjusting the **widths** of the components that may be affected by it. 

## Problems
* FIXED: The menu area stopped about four fifths of the way down the page.

@kathrynljackson
@taylorjohnson141
@aemcdonald

### CODE REVIEW

**Review by taylorjohnson141 — APPROVED** [review]
Looks great! I really love the changes you've made!

File: styles.css [hunk 1]
```diff
@@ -49,6 +49,7 @@ html {
   background: #1F1F3D;
   height: 100%;
   min-width: 200px;
+  position: fixed;
```
Comment (kathrynljackson) [inline styles.css:52]: Mentor suggested that we make the **.menu** a **fixed: position**.  We had to adjust the widths of several other components of the webpage, including in the media query.

**Review by kathrynljackson — COMMENTED** [review]
Coded together while discussing reasoning.

**Review by aemcdonald — APPROVED** [review]
code written together

**Pull request merged**